element_array_ephys/ephys_report.py

```diff
@@ -1,23 +1,23 @@
 import pathlib
 import datetime
 import datajoint as dj
-import typing as T
-import json
+from . import probe
 
 schema = dj.schema()
 
 ephys = None
 
 
 def activate(schema_name, ephys_schema_name, *, create_schema=True, create_tables=True):
+    """Activate the current schema.
+
+    Args:
+        schema_name (str): schema name on the database server to activate the `ephys_report` schema.
+        ephys_schema_name (str): schema name of the activated ephys element for which this ephys_report schema will be downstream from.
```

Comment:
not crucial, but I like to wrap these at 88 following our style convention elsewhere
```suggestion
        ephys_schema_name (str): schema name of the activated ephys element for which 
                this ephys_report schema will be downstream from.
```